core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,16 +18,57 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
+    @media(max-width: 768px) {
+      /* This prevents the navbar from collapsing in the exploration page,
+      so that the exploration title can be shown. */
+      .oppia-exploration-header {
+        display: block;
+        float: right;
+        max-width: 240px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+      }
+      .oppia-navbar-collapse.collapse {
+        display: block;
+      }
+    }
+    @media(max-width: 600px) {
+      .oppia-logo {
+        height: 48px;
+        margin-top: 8px;
+        margin-left: 0;
```

Comment:
Ah, if I am right `margin-top: 8px` should come after `margin-left: 0`, sorry I didn't noticed it. After this we'll be able to merge it.